src/main/java/aris/ui/Ui.java

```diff
@@ -0,0 +1,23 @@
+package aris.ui;
+
+public class Ui {
+    private String hline = "____________________________________________________________\n";
+    private String lastMessage = "";
+
+    public void format(String s) {
```

Comment:
```suggestion
    public void printFormattedLine(String s) {
```
Perhaps a name that better describes its behaviour?